roles/network_plugin/kube-ovn/templates/cni-ovn.yml.j2

```diff
@@ -463,7 +463,6 @@ spec:
 {% endif %}
       nodeSelector:
         kubernetes.io/os: "linux"
-        ovn.kubernetes.io/ovs_dp_type: "kernel"
```

Comment:
This line has been added since the commit f1d0d1a9fe76aeaa42e221e60f73da6408418e3f to update the kube-ovn version to v1.10.7.
The commit was created based on the latest kube-ovn manifest like https://raw.githubusercontent.com/kubeovn/kube-ovn/master/dist/images/install.sh

That contains the above line.
@liupeng0518 Could you take a look at this pull request?

Reply:
This install script adds the tag dependent on DPDK though and has an alternative DaemonSet for ovs_dp_type=userspace.

if [ "$DPDK" = "true" -o "$HYBRID_DPDK" = "true" ]; then
  kubectl label no -lovn.kubernetes.io/ovs_dp_type!=userspace ovn.kubernetes.io/ovs_dp_type=kernel --overwrite
fi

All this is all needed for HYBRID_DPDK, which kubespray doesn't support (yet?). 

In the standard case, this line is not present:
https://github.com/kubeovn/kube-ovn/blob/master/dist/images/install.sh#L2596

Reply:
Thanks for your patch, removing it makes it work for me as well! However, I was wondering whether instead of removing this line, it should be made dependent on `kube_ovn_dpdk_enabled` like the other settings in this file?

Reply:
> This line has been added since the commit [f1d0d1a](https://github.com/kubernetes-sigs/kubespray/commit/f1d0d1a9fe76aeaa42e221e60f73da6408418e3f) to update the kube-ovn version to v1.10.7. The commit was created based on the latest kube-ovn manifest like https://raw.githubusercontent.com/kubeovn/kube-ovn/master/dist/images/install.sh
> 
> That contains the above line. @liupeng0518 Could you take a look at this pull request?

Indeed, this is a mistake.

Reply:
@liupeng0518 Thank you for checking it.
It is fine to merge this pull request as is for me.

/approve